### ISSUE TITLE
#-1052927278------------- Elements must only use allowed ARIA attributes

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1858]
<strong>PageName:</strong> FeedViewDetailsPage  

<strong>Source:</strong>`<span class="share-summary__button flex flex-center flex-0" aria-label="Sharing level for feed_automationTest_httpPoller_JSON_2024_02_08_0403: Owner. " id="d4f81ae9d48741f795eeeafb200bfdfe-share-summary" aria-describedby="calcite-tooltip-9499f256-bea3-32d5-e44c-808d4eb0af8a">`

<strong>Element Location: </strong>#d4f81ae9d48741f795eeeafb200bfdfe-share-summary

<strong>Impact: </strong>serious

<strong>Failure Summary:</strong> Fix all of the following:
  aria-label attribute cannot be used on a span with no valid role attribute.

<strong>Tags:</strong> cat.aria,wcag2a,wcag412

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/aria-allowed-attr?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedViewDetailsPage\-1052927278_Elements must only use allowed ARIA attributes.png`
